app/js/locales/de-de.json

```diff
@@ -66,6 +66,7 @@
     "settings_modal_pushes": "Hintergrund-Benachrichtigungen",
     "settings_modal_message_preview": "Nachrichtenvorschau",
     "settings_modal_sound": "Ton",
+    "settings_modal_dark_mode": "Nacht-Modus",
```

Comment:
`"settings_modal_active_theme": "Aktiv-Thema",`

Might need to double check since I used Google Translate.